EIPS/eip-1.md

```diff
@@ -191,7 +191,7 @@ Links to external resources **SHOULD NOT** be included. External resources may d
 
 ## Linking to other EIPs
 
-References to other EIPs should follow the format `EIP-N` where `N` is the EIP number you are referring to.  Each EIP that is referenced in an EIP **MUST** be accompanied by a relative markdown link the first time it is referenced, and **MAY** be accompanied by a link on subsequent references.  The link **MUST** always be done via relative paths so that the links work in this GitHub repository, forks of this repository, the main EIPs site, mirrors of the main EIP site, etc.  For example, you would link to this EIP with `[EIP-1](./eip-1.md)`.
+References to other EIPs should follow the format `EIP-N` where `N` is the EIP number you are referring to.  Each EIP that is referenced in an EIP **MUST** be accompanied by a relative markdown link the first time it is referenced, and **MAY** be accompanied by a link on subsequent references.  The link **MUST** always be done via relative paths so that the links work in this GitHub repository, forks of this repository, the main EIPs site, mirrors of the main EIP site, etc.  For example, you would link to this EIP with `[EIP-1](.​/eip-1.md)`.
```

Comment:
What is the change here?  Neither GitHub nor my desktop compare tool tell me what changed, but both agree that the lines are different.  😖

Reply:
I added a zero width space between the `.` and the `/`:
<img width="119" alt="Screenshot 2022-07-12 081052" src="https://user-images.githubusercontent.com/45835846/178486682-8f2af013-17c9-4c03-89e5-e33cfa99f867.png">